src/py/flwr/driver/driver.py

```diff
@@ -126,3 +128,77 @@ def pull_task_res(self, req: PullTaskResRequest) -> PullTaskResResponse:
         # Call Driver API
         res: PullTaskResResponse = self.stub.PullTaskRes(request=req)
         return res
+
+
+class Driver:
+    """`Driver` class provides an interface to the Driver API."""
+
+    def __init__(self) -> None:
+        self.grpc_driver = GrpcDriver()
```

Comment:
Should we rather initialize this object in `_check_and_init_grpc_driver`?

Reply:
May I know the reasons for moving the init code in `_check_and_init_grpc_driver`?

There's actually a small problem after moving the code. `mypy` does not understand `_check_and_init_grpc_driver` method and hence it will raise many `Item "None" of "Optional[GrpcDriver]" has no attribute "xxxxx"` errors.